### DISCUSSION TITLE
Does Kong Manager support db-less and a read-only mode?

### DISCUSSION THREAD

**Discussion by mikaello** [Kong/kong-manager#74] [Q&A] [answered]
Is it possible to use Kong Manager as a read-only UI for a db-less instance? To get the possibility to view routes and plugins in a GUI would be very helpfull, even if it is not possible to change them.

**Comment by sumimakito** [comment, marked as answer]
Hello @mikaello, Kong Manager should display entities normally with a Gateway instance started in DB-less mode. According to [the documentation for DB-less mode](https://docs.konghq.com/gateway/latest/production/deployment-topologies/db-less-and-declarative-config/#read-only-admin-api), most of the features in Kong Manager will be read-only with restrictions applied to the Admin API.
